src/library/export/trackexportdlg.cpp

```diff
@@ -48,6 +52,36 @@ void TrackExportDlg::showEvent(QShowEvent* event) {
         qDebug() << "Programming error: did not initialize m_exporter, about to crash";
         return;
     }
+
+    //Checks if there are any files that are missing. If so, it will throw dialog informing the user
+    QList<QString> files = m_worker->getMissingTracks();
+    if (files.length() > 0) {
+        QDialog* notExportedDlg = new QDialog(nullptr);
+        notExportedDlg->setWindowTitle(tr("Export Track Files"));
+        QVBoxLayout* notExportedLayout = new QVBoxLayout;
+        QLabel* notExportedLabel = new QLabel;
+        notExportedLabel->setText(
+                tr("The following %1 files were not found at the specified "
+                   "file location and as a result were not exported")
+                        .arg(QString::number(files.length())));
+        notExportedLabel->setTextFormat(Qt::RichText);
+
+        QListWidget* notExportedList = new QListWidget;
+        notExportedList->addItems(files);
+
+        QDialogButtonBox* exportedDlgButtons = new QDialogButtonBox();
+        QPushButton* okBtn = exportedDlgButtons->addButton(
```

Comment:
Use https://doc.qt.io/qt-5/qdialogbuttonbox.html#addButton-2 to add a standard button